hooks/useExchange.ts

```diff
@@ -117,6 +118,7 @@ const useExchange = ({ showNoSynthsCard = false }: ExchangeCardProps) => {
 	const setHasOrdersNotification = useSetRecoilState(hasOrdersNotificationState);
 	const quoteCurrencyAmountBN = useRecoilValue(quoteCurrencyAmountBNState);
 	const baseCurrencyAmountBN = useRecoilValue(baseCurrencyAmountBNState);
+	useExchangeRatesQuery({ refetchInterval: 15000 });
```

Comment:
I know this is already merged but noticed it as looking through for something unrelated and I'm not sure we need this extra query do we, as it's already called in RefetchProvider which wraps the whole app?